setup/enginedialog.py

```diff
@@ -305,6 +305,7 @@ def __show_lang_rows(self):
     def __show_more(self):
         self.__set_fixed_size()
         self.__filter_entry.show()
```

Comment:
Could you add a comment to explain the line?
My suggestion "# Get focus after clicking on the view-more row."